Adding test report for One Click Install.

cybersecfaith opened the pull request:
Adding test report for One Click Install.

JanLahmann commented:
Hi @cybersecfaith , thank you so much for this extensive and thorough test report!
It will be a great help for RasQberry users. 

Some comments:
* Should we add it to the gh-pages brauch? (Plus s remark in master / Readme.md) There are some conflicts against the master branch.
* instead of getgist, we now use wget. See readme.md in master
* Did you execute any qiskit code locally? E.g. one of the notebooks in fun—with-quantum? I’m not sure if the qiskit install might be corrupt. 
* Did you also execute „one click demo install“? This is needed for e.g. the Qoffee-Maker
* I will comment later on all of the failed items (and if we can fix some of them)

cybersecfaith commented:
Hi @JanLahmann - My error. I should have pushed it to gh-pages. No wonder all the conflicts in master. I have changed the base branch to gh-pages now and the merge should go through as I do not see conflicts on my end. 

To answer your questions:

- [X] Should we add it to the gh-pages brauch? (Plus s remark in master / Readme.md) There are some conflicts against the master branch. - STATUS: DONE
- [X] instead of getgist, we now use wget. See readme.md in master - I actually saw both options in the documentation BUT did not know wget was the newer solution so wget would be the better option. I've added that to Step 3 under option A - STATUS: DONE
- [ ] Did you execute any qiskit code locally? E.g. one of the notebooks in fun—with-quantum? I’m not sure if the qiskit install might be corrupt. - No I am yet to run any of the notebooks. STATUS: To-DO
- [ ] Did you also execute „one click demo install“? This is needed for e.g. the Qoffee-Maker - Not yet. I'll run that next. This push was just for the one cliick install part.  STATUS: To-DO
    
Based on your comment, my next step will be to run the one click demo install and then rerun the tests above to see if some of the errors are fixed. 

Ps. There should be no conflicts now, Please review and merge Part 1.
